src/class-parsely.php

```diff
@@ -92,6 +93,24 @@ public function __construct() {
 		add_action( 'wp_enqueue_scripts', [ $this, 'wp_parsely_style_init' ] );
 	}
 
+	function add_parsely_link( $actions, $post ) {
+		if ( ! self::post_has_trackable_status( $post ) ) {
+			return $actions;
+		}
```

Comment:
While a post may not currently have a trackable status, it may have had one previously - and so there may be data in Parse.ly dashboard about it.

Is this an edge case that we should ignore for now?

Reply:
Yes, I think this is an ok edge case to ignore for now. If it becomes obvious how we can know if an article has been crawled, we can revisit.  I think we're going to want to do so if / when we make this indication more dynamic.